radixdlt/src/main/java/com/radixdlt/consensus/safety/SafetyRules.java

```diff
@@ -69,30 +113,30 @@ public void process(QuorumCertificate qc) {
 	 * @return A vote result containing the vote and any committed vertices
 	 * @throws SafetyViolationException In case the vertex would violate a safety invariant
 	 */
-	public VoteResult voteFor(Vertex proposedVertex) throws SafetyViolationException {
-		// ensure vertex does not violate earlier rounds
-		if (proposedVertex.getRound().compareTo(this.state.lastVotedRound) <= 0) {
+	public Vote voteFor(Vertex proposedVertex) throws SafetyViolationException, CryptoException {
```

Comment:
CryptoException here doesn't seem right.

Reply:
Hmm, debated back and forth on this one, but happy to change it back.